Samples/PrivateDns/ManagePrivateDns.cs

```diff
@@ -0,0 +1,197 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Management.Compute.Fluent;
+using Microsoft.Azure.Management.Compute.Fluent.Models;
+using Microsoft.Azure.Management.Fluent;
+using Microsoft.Azure.Management.Network.Fluent;
+using Microsoft.Azure.Management.ResourceManager.Fluent;
+using Microsoft.Azure.Management.ResourceManager.Fluent.Core;
+using Microsoft.Azure.Management.Samples.Common;
+using System;
+using System.Collections.Generic;
+
+namespace ManagePrivateDns
+{
+    public class Program
+    {
+        private const string CustomDomainName = "private.contoso.com";
+
+        /**
+         * Azure Private DNS sample for managing DNS zones.
+         *  - Creates a Private DNS Zone (private.contoso.com)
+         *  - Creates a Virtual Network
+         *  - Link a Virtual Network
+         *  - Creates test Virtual Machines
+         *  - Creates an additional DNS Record
+         *  - Test the private DNS zone
+         */
+        public static void RunSample(IAzure azure)
+        {
+            string rgName = SdkContext.RandomResourceName("MyARG", 24);
+            string vnName = SdkContext.RandomResourceName("MyVN", 24);
+            string subnetName = SdkContext.RandomResourceName("BackendSubnet", 24);
+            string linkName = SdkContext.RandomResourceName("MyVNLINK", 24);
+            string vm1Name = SdkContext.RandomResourceName("MyVM01", 24);
+            string vm2Name = SdkContext.RandomResourceName("MyVM02", 24);
+            string rsName = SdkContext.RandomResourceName("MyRecordSet", 24);
+            string osDiskName = SdkContext.RandomResourceName("OSDisk", 24);
+
+            try
+            {
+                var resourceGroup = azure.ResourceGroups.Define(rgName)
+                    .WithRegion(Region.AsiaSouthEast)
+                    .Create();
+
+                //============================================================
+                // Creates a Private DNS Zone
+
+                Utilities.Log("Creating private DNS zone " + CustomDomainName + "...");
+                var privateDnsZone = azure.PrivateDnsZones.Define(CustomDomainName)
+                    .WithExistingResourceGroup(resourceGroup)
+                    .Create();
+
+                Utilities.Log("Created private DNS zone " + privateDnsZone.Name);
+                Utilities.Print(privateDnsZone);
+
+
+                //============================================================
```

Comment:
Unless is proper noun, don't use capitalize term

Reply:
Done for the change